front/lib/api/assistant/global_agents.ts

```diff
@@ -793,6 +868,18 @@ async function _getDustGlobalAgent(
   const description = "An assistant with context on your company data.";
   const pictureUrl = "https://dust.tt/static/systemavatar/dust_avatar_full.png";
 
+  const model: AgentModelConfigurationType = !auth.isUpgraded()
```

Comment:
It's in a different function and I did not bother factorize it since 1/ This was already not factorized before and 2/ I'm hoping we can get rid of this soon (since it's only for former workspaces that still have messages available in their 50 messages).